svc-bip-api/src/main/java/gov/va/vro/bip/service/BipApiService.java

```diff
@@ -188,19 +183,19 @@ public BipUpdateClaimResp updateClaimContention(long claimId, UpdateContentionRe
     }
   }
 
-  @Override
-  public boolean confirmCanCallSpecialIssueTypes() {
-    String url = HTTPS + bipApiProps.getClaimBaseUrl() + SPECIAL_ISSUE_TYPES;
-    log.info("Call {} to get special_issue_types", url);
-
-    HttpHeaders headers = getBipHeader();
-    HttpEntity<Void> httpEntity = new HttpEntity<>(headers);
-
-    ResponseEntity<String> response =
-        restTemplate.exchange(url, HttpMethod.GET, httpEntity, String.class);
-
-    return response.getStatusCode() == HttpStatus.OK && !response.getBody().isEmpty();
-  }
+  //  @Override
+  //  public boolean confirmCanCallSpecialIssueTypes() {
+  //    String url = HTTPS + bipApiProps.getClaimBaseUrl() + SPECIAL_ISSUE_TYPES;
+  //    log.info("Call {} to get special_issue_types", url);
+  //
+  //    HttpHeaders headers = getBipHeader();
+  //    HttpEntity<Void> httpEntity = new HttpEntity<>(headers);
+  //
```

Comment:
Should we leave a comment to describe why this is commented out for now rather than being deleted and what conditions would result in this needing to be uncommented?

Reply:
This code was only used in RRD tests, I refactored it to make its purpose clear and gave it tests